Version 1.1.0 release updates

timmyc opened the pull request:
Prepping for the release of v1.1.0

timmyc committed:
Version 1.1.0 release updates

psealock reviewed (approved):
Changes look good to me ✅

jeffstieler commented on readme.txt:
```diff
@@ -71,6 +71,49 @@ Release and roadmap notes are available on the [WooCommerce Developers Blog](htt
 
+= 1.1.0 TBD =
+- Tweak: Update form example to better format the form return data #3706
+- Dev: Align WooCommerce javascript code standards/linting/styles with WordPress core. #3674
```
Omit this?

jeffstieler commented on readme.txt:
```diff
@@ -71,6 +71,49 @@ Release and roadmap notes are available on the [WooCommerce Developers Blog](htt
 == Changelog ==
 
+= 1.1.0 TBD =
+- Tweak: Update form example to better format the form return data #3706
```
Omit this? It's only in the DevDocs..

jeffstieler commented on readme.txt:
```diff
@@ -71,6 +71,49 @@ Release and roadmap notes are available on the [WooCommerce Developers Blog](htt
+- Tweak: Added link to "go shopping" button #3712
+- Fix: Make analytics tables use the site's date format setting #3715
+- Fix: Alignment of select text #3723 🎉 edmundcwm
+- Dev: Added Storybook for documenting internal components. #3679
```
Omit?

jeffstieler commented on readme.txt:
```diff
@@ -71,6 +71,49 @@ Release and roadmap notes are available on the [WooCommerce Developers Blog](htt
+- Dev: Add PayFast payment gateway option for sites in South Africa #3738 
+- Dev: Onboarding: Update screen order and remove Jetpack connection in profiler #3739
+- Dev: Onboarding - business step: add more options in the competitors list and other fixes #3812
+- Dev: Onboarding: Redesign plugin benefits screen #3764
```
These 4 lines aren't actually "Dev". They are either "Enhancement" or "Tweak".

jeffstieler commented on readme.txt:
```diff
@@ -71,6 +71,49 @@ Release and roadmap notes are available on the [WooCommerce Developers Blog](htt
+- Tweak: Remove 'add first product' note. #3876
+- Enhancement: Improve focus on task list #3796
+- Enhancement: Allow individual payment method setup in the onboarding task list #3782
+- Dev: Onboarding: Add toggles to configured payments in task list #3801
```
Again, not dev.

jeffstieler commented on readme.txt:
```diff
@@ -71,6 +71,49 @@ Release and roadmap notes are available on the [WooCommerce Developers Blog](htt
+- Enahncement: Add WC Pay to startup checklist
+- Enhancement: Marketing Tab
+- Enhancement: Add WooCommerce Payments Inbox note #4013
+- Dev: Added hook to delete woocommerce_onboarding_homepage_post_id #4015
```
Not dev, probably "fix"?

octaedro reviewed (approved):
It's testing well here :shipit:

Commit:
Enable wcpay feature flag in plugin and core

psealock committed:
Dynamic Currency with Context API (#4027)

* Make currency setting dynamic with context API

* reference Currency

* what is 'this'

* remove comment

* All report tables

* Customers report

* activity panel

* leaderboard test

* business details

* store details

* single source of truth

* shipping rates

* remove currency-format.js

* tidy up

* make Currency a factory function

* factory revenue table

* tables

* Chart y-axis labels

* customers table

* cleanup

* business details

* shipping

* number-format

* stock and downloads tables

* changelog

* cleanup

* SQL example: update to dynamic currencies (#4033)

jameskoster committed:
Replace welcome image on marketing tab with svg. closes #4057 (#4058)

octaedro committed:
Bugfix on payments step in CBD flow

The `array_search` method was changed to `in_array`

octaedro committed:
After pressing Connect a new tab opens with the Square webpage

danbitzer committed:
Set a max word length for the installed extension descriptions

danbitzer committed:
Improve InstalledExtensions layout for tablet and mobile

Mobile still needs work…

joshuatf committed:
Fix double prefixed wcadmin events

octaedro committed:
Updated behavior for non CBD Square payment connection

timmyc committed:
Update Jetpack Autoloader.

jeffstieler committed:
OBW: fix use of unbound this in business details step. (#4070)

timmyc committed:
Setup Checklist: Fix exception in shipping.

octaedro committed:
Square option's name updated

octaedro committed:
Replaced <p> tag with a <div> tag

Since there was a `validateDOMNesting` warning because we weren't adding only text inside a <p> tag, the `p` tag was changed to a `div`.

timmyc committed:
Remove menu item

timmyc committed:
Remove 'do this later' link

timmyc commented:
@jeffstieler is it okay if I merge this as-is, then apply those readme changes in a follow-up PR? Now that I've been cherry-picking some fixes to `release\1.1.0` I fear that things might get tricky with the branch.

rrennick committed:
remove use of undefined QUEUE_GROUP (#3994)

* remove use of undefined QUEUE_GROUP

* remove unsnooze action from both queue groups

findingsimple committed:
Add missing prompts to installed extensions (#4060)

* Add support for additional partner extensions to installed extensions

* Use correct variable name

* Add codisto support url

* Fix hubspot svg

- Fixes orange color as per brand guide
- Reduces file size from 120kb to 1kb

* Replace Google logo with vector SVG

* Fix AutomateWoo icon

* Fix Amazon Ebay svg icon

Co-authored-by: Daniel Bitzer <danielbitzer@gmail.com>

jeffstieler commented:
> is it okay if I merge this as-is, then apply those readme changes in a follow-up PR?

With the cherry picking, I don't think we need to (or should) merge this branch into `master`. While this comparison shows a diff, these commits are already on the `master` branch: https://github.com/woocommerce/woocommerce-admin/commits/master

octaedro committed:
Added check to show WC Pay option only when Jetpack is connected (#4105)

* Added check to show WC Pay option only when Jetpack is connected

A verification was added to the Setup Checklist payment step, to show WC Pay option only when Jetpack is connected

* Changed the solution approach

Co-authored-by: Fernando Marichal <contacto@fernandomarichal.com>

psealock commented:
>With the cherry picking, I don't think we need to (or should) merge this branch into master. While this comparison shows a diff, these commits are already on the master branch: https://github.com/woocommerce/woocommerce-admin/commits/master

That works for me, but the version bump and changelog changes need to make their way into master.

jeffstieler committed:
Handle orphaned order statuses in analytics settings. (#4090)

* Add method to retrieve all synced order statuses.

* Determine what unregistered order statuses we have and add to client-side settings object.

* Stop filtering out unregistered statuses from analytics settings.

* Add unregistered statuses to analytics settings fields.

Commit:
Overwrite button overflow css from Wordpress 5.4 defaults (#4108)

jeffstieler committed:
Fix usage of WP_Error in non-global namespaces. (#4115)

octaedro committed:
Onboarding - business details step: increase max-height on competitors listbox (#4111)

* Enlarged competitors' Listbox in the Onboarding

Onboarding - Added a new class to enlarge the competitors' Listbox in the Business details step.

* Replaced fixed height with unset

Co-authored-by: Fernando Marichal <contacto@fernandomarichal.com>

psealock committed:
OBW: allow CBD only for US stores (#4117)

* OBW: allow CBD only for US stores

* fix if user goes back

* industry.js feedback

* store-details feedback

* better naming

psealock committed:
changelog entries

psealock committed:
Store details: check if industries exist before trimming

danbitzer committed:
Align product icon to top

danbitzer committed:
Tweak WelcomeCard mobile appearance

danbitzer committed:
Fix alignment

timmyc commented:
Closing in favor of #4158
